atom/browser/native_window_mac.mm

```diff
@@ -719,6 +706,34 @@ - (void)drawRect:(NSRect)dirtyRect {
   return [window_ styleMask] & NSResizableWindowMask;
 }
 
+void NativeWindowMac::SetAspectRatio(
+    double aspect_ratio, const gfx::Size& extra_size) {
+
```

Comment:
I just went with the existing convention of the file. This for example: https://github.com/electron/electron/blob/master/atom/browser/native_window_mac.mm#L687-688